.gitignore

```diff
@@ -65,3 +65,6 @@ typings/
 test2.js
 
 .vscode
+
+package-lock.json
+yarn.lock
```

Comment:
[Do not ignore yarn.lock](https://github.com/yarnpkg/yarn/issues/1583#issuecomment-257544217)